src/client/components/builder/questions/RadioField.tsx

```diff
@@ -101,60 +111,76 @@ const InputComponent: QuestionFieldComponent = ({ field, index }) => {
-            name="description"
-            placeholder="Description"
-            onChange={updateTitleOrDescription}
-            value={description}
-          />
-        </VStack>
-        <VStack spacing={4} alignItems="left" w="50%">
-          {field.options.map(renderOption)}
-          <HStack h={10}>
-            <Radio isChecked={false} />
-            <Box pl={2}>
-              <Button variant="link" onClick={addOption}>
-                Add new option
-              </Button>
-            </Box>
-          </HStack>
-        </VStack>
+    <VStack sx={commonStyles.fullWidthContainer} spacing={4}>
+      <InputGroup>
+        <InputLeftElement
+          sx={commonStyles.inputIconElement}
+          children={<BiRadioCircleMarked />}
+        />
+        <Input
+          type="text"
+          sx={commonStyles.fieldInput}
+          name="title"
+          placeholder="Question"
+          onChange={updateTitleOrDescription}
+          value={title}
+        />
+      </InputGroup>
+      <Input
+        type="text"
+        sx={commonStyles.fieldInput}
+        name="description"
+        placeholder="Description"
+        onChange={updateTitleOrDescription}
+        value={description}
+      />
+      <VStack sx={commonStyles.halfWidthContainer} spacing={4}>
+        {field.options.map(renderOption)}
+        <HStack sx={styles.addOptionContainer} spacing={4}>
+          <Icon as={BiPlus} sx={styles.addOptionIcon} />
+          <Button
+            variant="link"
+            sx={styles.addOptionButton}
+            onClick={addOption}
+          >
+            Add option
+          </Button>
+        </HStack>
       </VStack>
-    </HStack>
+    </VStack>
   )
 }
 
-const PreviewComponent: QuestionFieldComponent = ({ field }) => {
+const PreviewComponent: QuestionFieldComponent = ({ field, index }) => {
   const { title, description, options } = field
+  const commonStyles = useStyles()
+  const styles = useMultiStyleConfig('RadioField', {})
+
   return (
-    <VStack align="stretch" w="100%" spacing={6}>
-      <VStack align="stretch">
+    <VStack sx={commonStyles.fullWidthContainer} spacing={2}>
+      <VStack sx={commonStyles.fullWidthContainer} spacing={0}>
         <HStack>
-          <BiListUl fontSize="20px" />
-          <Text>{title}</Text>
+          <FieldIndexText index={index} />
+          <Text sx={commonStyles.previewTitle}>{title}</Text>
         </HStack>
-        {description && <Text color="secondary.400">{description}</Text>}
+        {description && (
+          <Text sx={commonStyles.previewDescription}>{description}</Text>
+        )}
       </VStack>
       <RadioGroup>
-        <VStack alignItems="left" spacing={2}>
+        <VStack sx={styles.previewOptionsContainer} spacing={0}>
           {options.map(({ value, label }, i) => (
-            <Radio key={i} value={value} isChecked={false}>
-              {label}
-            </Radio>
+            <Flex sx={styles.previewOptionRowContainer}>
```

Comment:
```suggestion
            <Flex key={i} sx={styles.previewOptionRowContainer}>
```

Key should be on the top-level element to avoid warning from React.